### ISSUE TITLE
please add support for "325" and "375"

### ISSUE THREAD

**Issue by PixxelSong** [flaneur2020/pua-lang#8]

**Comment by ICEYSELF** [comment]
We can use integral number `325` to represent boolean `false`, and `375` for `true`, I think.

**Comment by chenhengqi** [comment]
As you mentioned in #1 , we should use `三七五` and `三二五` instead.

**Comment by ICEYSELF** [comment]
> As you mentioned in #1 , maybe we should use `三七五` and `三二五` instead.

Nope. We can handle this stuff at run-time instead of compile time, making these numbers equivalent to false and true in bool type. In fact this is what C/C++/JS do, while the only difference is these languages use zero and non-zero values.

**Comment by chenhengqi** [comment]
It's not strong-typed.
So in this case, we just need to treat `325` as a false value.

**Comment by ICEYSELF** [comment]
> It's not strong-typed.
> So in this case, we just need to treat `325` as a false value. WDYT?

No, because there are also `3` and `2.5` ;)

**Comment by chenhengqi** [comment]
🤦
